bedrock/products/templates/products/vpn/landing.html

```diff
@@ -199,14 +203,50 @@ <h3>{{ ftl('vpn-landing-faq-competition-question-heading') }}</h3>
       <summary class="vpn-faq-item-heading">
         <h3>{{ ftl('vpn-landing-faq-compatibility-question-heading') }}</h3>
       </summary>
-      <p class="vpn-faq-item-detail">{{ ftl('vpn-landing-faq-compatibility-question-desc') }}</p>
+      <p class="vpn-faq-item-detail">
+        {% if ftl_has_messages('vpn-landing-faq-compatibility-question-desc-v2') %}
+          {{ ftl('vpn-landing-faq-compatibility-question-desc-v2', mobile=url('products.vpn.platforms.mobile'), desktop=url('products.vpn.platforms.desktop')) }}
+        {% else %}
+          {{ ftl('vpn-landing-faq-compatibility-question-desc') }}
+        {% endif %}
+      </p>
 
       <ul class="vpn-faq-item-detail mzp-u-list-styled">
-        <li>{{ ftl('vpn-landing-faq-compatibility-question-desc-windows') }}</li>
-        <li>{{ ftl('vpn-landing-faq-compatibility-question-desc-mac') }}</li>
-        <li>{{ ftl('vpn-landing-faq-compatibility-question-desc-android') }}</li>
-        <li>{{ ftl('vpn-landing-faq-compatibility-question-desc-ios') }}</li>
-        <li>{{ ftl('vpn-landing-faq-compatibility-question-desc-linux') }}</li>
+        <li>
```

Comment:
Copy question: Is this order intentional?
- Windows 10
- macOS
- Android
- iOS
- Linux

Reply:
By popularity yes!